tests/set1.py

```diff
@@ -0,0 +1,9 @@
+def test_Set():
+    a: i32
+    b: i32
+
+    a = {1, 2, 3}
+    a = {2, 3, 4, 5, 5}
+    a = {"a", "b", "c"}
```

Comment:
All these should give an error message that "Set[i32] (or character) cannot be assigned to i32". 

Then let's add tests like:
```
a: Set[i32]
a = {1, 2, 3}
```
And we need to improve the assignment to do this check.